pyproject.toml

```diff
@@ -1,3 +1,54 @@
+[project]
+name = "ops"
+description = "The Python library behind great charms"
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+    {name="The Charm Tech team at Canonical Ltd.", email="charm-tech@lists.launchpad.com"},
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+]
+dependencies = [
+    "PyYAML==6.*",
+    "websocket-client==1.*",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+docs = [
+    "sphinx==6.2.1",
+    "sphinx-design",
+    "furo",
+    "sphinx-tabs",
+    "lxd-sphinx-extensions",
+    "sphinx-copybutton",
+]
+
+[project.urls]
+"Homepage" = "https://juju.is/docs/sdk"
+"Repository" = "https://github.com/canonical/operator"
+"Issues" = "https://github.com/canonical/operator/issues"
+"Documentation" = "https://ops.readthedocs.io"
+"Changelog" = "https://github.com/canonical/operator/blob/main/CHANGES.md"
+
+[build-system]
+requires = [
+    "setuptools>=60",
+    "PyYAML==6.*",
+    "websocket-client==1.*",
```

Comment:
Can we remove these two now that we're using `attr=ops.version.version`, which is avoiding the import?
```suggestion
```

Reply:
Ah, thanks, I indeed forgot to do that part.